src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs

```diff
@@ -13,78 +14,78 @@ public class DbConfig : IDbConfig
     public uint WriteBufferNumber { get; set; } = 4;
     public ulong BlockCacheSize { get; set; } = (ulong)64.MiB();
     public bool CacheIndexAndFilterBlocks { get; set; } = false;
-    public int? MaxOpenFiles { get; set; }
+    public int? MaxOpenFiles { get; set; } = -1;
     public long? MaxWriteBytesPerSec { get; set; }
 
     public ulong ReceiptsDbWriteBufferSize { get; set; } = (ulong)8.MiB();
     public uint ReceiptsDbWriteBufferNumber { get; set; } = 4;
     public ulong ReceiptsDbBlockCacheSize { get; set; } = (ulong)32.MiB();
     public bool ReceiptsDbCacheIndexAndFilterBlocks { get; set; } = false;
-    public int? ReceiptsDbMaxOpenFiles { get; set; }
+    public int? ReceiptsDbMaxOpenFiles { get; set; } = -1;
```

Comment:
I think you should leave this null so that if you want to specify max open files for all db, you only need to use the general one.

Reply:
Already resolved, thank you=)